app/posts/register-trainee-teachers/2022-09-02-updating-our-support-user-interface.md

```diff
@@ -0,0 +1,107 @@
+
+The first change is to have a single navigation bar when signed in as a support user - instead of two competing UIs we currently have. This should make it easier to navigate to the relevant sections and introduce more of a distinction between the views regular users have and the views admins have.
+
+### Support index page
+
+![Screenshot of new support home page. There are several tiles on the page each showing a metric and count, such as number of users, or number of trainees.](3.support-index.png)
+
+Support index is a home page which can contain useful stats and links to common items.
+
+### Users index
+
+![Screenshot of a page for listing all registered users. On the left are some checkboxes for selecting organisation type. On the right are a list of users and a autocomplete search box.](4.users-index.png)
+
+The users index lists all users in the service, with filters to filter by type, and an autocomplete to search for users using their name, organisation, or email address.
+
+![A screenshot of the autocomplete with a text search of "jane". There is one result for a user.](5.users-autocomplete.png)
+
+The autocomplete has been enhanced to directly go to the user when one is selected.
+
+### User page
+
+![A screenshot of page showing details about a user. The page has several tables of data, some of which that have change links.](6.user-page.png)
+
+The user page shows information about Register users, with links to change relevant fields. It also shows the current organisations the user is connected to with the ability to add the user to additional organisations, remove them, or change their access rights.
+
+#### Adding a user to an organisation
+
+![Screenshot of a page for adding a user to an organisation. The page asks the user for the organisation name and ](7.adding-user-to-organisation.png)
+
+When adding a user to an organisation, users get an autocomplete of available organisations and additional questions about the user’s role.
+
+We have yet to build granular user permissions in Register, but this design explores how we might approach it when we do.
+
+The new designs also support starting with an organisation and choosing to add a user to it. Which approach to use might vary depending on the task - if a support agent has several users to add to one org, they might start with that org. If they have an existing user they need to add to a second org, it might make more sense to start with the user.
+
+#### Changing a user’s access to an organisation
+
+![Screenshot of a page for editing a user’s access to an organisation. There are radio options for role type, and to remove access or archive the user.](8.user-organisation-access.png)
+
+For each organisation a user has access to, we can have an edit page to manage that relationship. Initially this would likely just be to remove access to an org - but in the future could control the user’s permissions at that organisation.
```

Comment:
```suggestion
For each organisation a user has access to, we can have an edit page to manage that relationship. Initially this would likely just be to remove access to an organisation - but in the future could control the user’s permissions at that organisation.
```